statsmodels/miscmodels/try_mlecov.py

```diff
@@ -112,18 +114,9 @@ def mvn_nloglike_obs(x, sigma):
     return llike
 
 def invertibleroots(ma):
-    import numpy.polynomial as poly
-    pr = poly.polyroots(ma)
-    insideroots = np.abs(pr)<1
-    if insideroots.any():
-        pr[np.abs(pr)<1] = 1./pr[np.abs(pr)<1]
-        pnew = poly.Polynomial.fromroots(pr)
-        mainv = pn.coef/pnew.coef[0]
```

Comment:
`pn` is undefined.  Could reasonably be typo for either `pr` or `pnew`.  Turns out to be `pnew`.  Also turns out we have another strictly-dominating option that does exactly the same thing.